test/remote-method.test.js

```diff
@@ -74,10 +73,7 @@ describe('loopback:remote-method generator', function() {
       }).then(function() {
         var definition = common.readJsonSync('common/models/car.json');
         var methods = definition.methods || {};
-        expect(methods).to.have.property('myRemote');
```

Comment:
Reading from this docs page: https://loopback.io/doc/en/lb3/3.0-Release-Notes.html#name-indicates-whether-method-is-static, I think this line and the line below are no longer true.  But I'm not 100% sure.  
@bajtos @jannyHou @hacksparrow, I'd like to get your inputs on this.  If this is true, perhaps we need to change the test name as well.

Test name `method name with 'prototype.' should be removed` -> `method name with 'prototype.' should not removed`?

Reply:
Your fix looks correct to me :+1: I agree to update the test name to match the actual assertions.

Reply:
@dhmlau Fair enough 💯